sky/cli.py

```diff
@@ -1450,40 +1483,65 @@ def status(all: bool, refresh: bool, clusters: List[str]):  # pylint: disable=re
+
+        if clusters:
+            clusters = _get_glob_clusters(clusters)
         else:
-            nonreserved_cluster_records.append(cluster_record)
-    local_clusters = onprem_utils.check_and_get_local_clusters(
-        suppress_error=True)
+            clusters = None
+        cluster_records = core.status(cluster_names=clusters, refresh=refresh)
+        nonreserved_cluster_records = []
+        reserved_clusters = []
+        for cluster_record in cluster_records:
+            cluster_name = cluster_record['name']
+            if cluster_name in backend_utils.SKY_RESERVED_CLUSTER_NAMES:
+                reserved_clusters.append(cluster_record)
+            else:
+                nonreserved_cluster_records.append(cluster_record)
+        local_clusters = onprem_utils.check_and_get_local_clusters(
+            suppress_error=True)
 
-    num_pending_autostop = 0
-    num_pending_autostop += status_utils.show_status_table(
-        nonreserved_cluster_records, all)
-    for cluster_group_name, cluster_record in reserved_clusters.items():
+        num_pending_autostop = 0
         num_pending_autostop += status_utils.show_status_table(
-            [cluster_record], all, reserved_group_name=cluster_group_name)
-    if num_pending_autostop > 0:
-        plural = ' has'
-        if num_pending_autostop > 1:
-            plural = 's have'
-        click.echo('\n'
-                   f'{num_pending_autostop} cluster{plural} '
-                   'auto{stop,down} scheduled. Refresh statuses with: '
-                   f'{colorama.Style.BRIGHT}sky status --refresh'
-                   f'{colorama.Style.RESET_ALL}')
-    status_utils.show_local_status_table(local_clusters)
+            nonreserved_cluster_records + reserved_clusters, all)
+        status_utils.show_local_status_table(local_clusters)
+
+        hints = []
+        if show_spot_queue:
+            click.echo(f'\n{colorama.Fore.CYAN}{colorama.Style.BRIGHT}'
+                       f'Managed Spot Jobs{colorama.Style.RESET_ALL}')
+            with backend_utils.safe_console_status(
+                    '[cyan]Checking spot jobs[/]'):
+                msg = spot_jobs_future.get()
+                try:
+                    pool.close()
+                    pool.join()
+                except SystemExit as e:
+                    # This is to avoid a "Exception ignored" problem caused by
+                    # ray worker setting the sigterm handler to sys.exit(15)
+                    # (see ray/_private/worker.py).
+                    if e.code != 15:
+                        raise
+            if msg:
+                click.echo(msg)
+                hints.append(f'* Use spot jobs CLI: {colorama.Style.BRIGHT}'
+                             f'sky spot --help{colorama.Style.RESET_ALL}; '
+                             f'Check all spot jobs: {colorama.Style.BRIGHT}'
+                             f'sky spot queue{colorama.Style.RESET_ALL}')
```

Comment:
Wdyt about
```
* 12 spot jobs are in progress (5 latest ones shown above). To see all jobs: sky spot queue
```
or if we aren't omitting any:
```
* 3 spot jobs are in progress. To see all jobs: sky spot queue
```
It seems okay to omit "Use spot jobs CLI: sky spot --help" to keep it shorter.

Reply:
Q about the `In progress jobs: 3 RUNNING, 2 STARTING` line in:
```
» sst
Clusters
NAME                          LAUNCHED     RESOURCES                            STATUS  AUTOSTOP  COMMAND
sky-spot-controller-8a3968f2  52 secs ago  1x GCP(n2-standard-4, disk_size=50)  INIT    10m       sky spot launch -dy --cloud...

Managed Spot Jobs
In progress jobs: 3 RUNNING, 2 STARTING
ID  NAME               RESOURCES     SUBMITTED   TOT. DURATION  JOB DURATION  #RECOVERIES  STATUS
19  sky-79c1-zongheng  1x [CPU:0.5]  1 min ago   1m             -             0            STARTING
18  sky-c2c8-zongheng  1x [CPU:0.5]  2 mins ago  2m 9s          -             0            STARTING
17  sky-a7fb-zongheng  1x [CPU:0.5]  3 mins ago  3m 21s         1m 21s        0            RUNNING
16  sky-d0b7-zongheng  1x [CPU:0.5]  4 mins ago  4m 32s         2m 33s        0            RUNNING
15  sky-a0e3-zongheng  1x [CPU:0.5]  5 mins ago  5m 44s         3m 40s        0            RUNNING

* Use spot jobs CLI: sky spot --help; Check all spot jobs: sky spot queue
* 1 cluster has auto{stop,down} scheduled. Refresh statuses with: sky status --refresh
```

Will that line be a breakdown of the top 5 in-progress jobs, or will it show all in progress jobs?

Ah, it seems like it's the former. That may be a bit confusing. Is it possible to omit that line in `status` output?